features/upgrade/machines/machines.feature

```diff
@@ -39,6 +39,18 @@ Feature: Machine-api components upgrade tests
     | "cluster-autoscaler"       | # @case_id OCP-27664
 
 
+  @upgrade-prepare
+  @admin
+  @4.10 @4.9
+  @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
+  @vsphere-upi @openstack-upi @gcp-upi @azure-upi @aws-upi
+  Scenario Outline: Cluster operator should be available after upgrade - prepare
+    Given the expression should be true> "True" == "True"
+
+  Examples:
```

Comment:
@huali9 Since this did a fake test, you can remove the examples section, and use `Scenario` instead of `Scenario Outline`.
Everything else looks good

Reply:
ok, will update it.

Reply:
updated.